pkg/embeddedetcd/config.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The KCP Authors.
+Copyright 2022 The KCP Authors.
```

Comment:
Do we have to do this?

Reply:
it's technically a new file, only Github shows it as rename.